Online/Menus/CSS/InitSheikSelector.asm

```diff
@@ -249,6 +249,15 @@ lbz r3, MSRB_IS_LOCAL_PLAYER_READY(r3)
 cmpwi r3, 0
 bne FN_SZThink_CHANGE_HANDLING_END # No changes when locked-in
 
+# Check if the Z button was pressed this frame
+load r4, 0x804c20bc
+lbz r3, -0x49B0(r13) # player index
+mulli r3, r3, 68
```

Comment:
nitpick: use a constant like PLAYER_PAD_STRUCT_LEN or something (idk exactly what struct youre looking at) instead of a magic number